src/main/java/com/slimebot/commands/WordchainCommand.java

```diff
@@ -0,0 +1,66 @@
+package com.slimebot.commands;
+
+import com.slimebot.games.Game;
+import com.slimebot.games.PlayerGameState;
+import com.slimebot.games.games.Wordchain;
+import de.mineking.discord.commands.annotated.ApplicationCommand;
+import de.mineking.discord.commands.annotated.ApplicationCommandMethod;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.emoji.Emoji;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+import net.dv8tion.jda.api.interactions.components.buttons.Button;
+import net.dv8tion.jda.api.interactions.components.selections.SelectOption;
+import net.dv8tion.jda.api.interactions.components.selections.StringSelectMenu;
+
+import java.time.Instant;
+
+
+@ApplicationCommand(name = "wordchain", description = "Spiele das Wordchain minispiel mit anderen Leuten!")
+public class WordchainCommand extends ListenerAdapter {
+    @ApplicationCommandMethod
+    public void performCommand(SlashCommandInteractionEvent event) {
+        Long memberId = event.getMember().getIdLong();
+        if(PlayerGameState.isInGame(memberId)) {
+            event.reply("Du bist schon in einem Game!").setEphemeral(true).queue();
+            return;
+        }
+
+        Game game = new Wordchain(memberId, event.getChannel(), event.getGuild().getIdLong());
```

Comment:
You should probably only create the game when the configuration is finished and then start the game with the configuration.